common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -720,6 +720,11 @@ public static String computeMaxHostBrokerProtocol() {
          */
         public static final String BROKER_REDIRECT_URI = "urn:ietf:wg:oauth:2.0:oob";
 
+        /**
+         * String for new broker redirect URI
```

Comment:
```suggestion
         * String for new broker redirect URI.
```

Reply:
We also have such constant in broker repo. Perhaps that one should just reference this one now.